proj4/src/main/scala/geotrellis/proj4/io/wkt/WKTParser.scala

```diff
@@ -0,0 +1,152 @@
+package geotrellis.proj4.io.wkt
+
+import scala.io.Source
+import geotrellis.proj4.Memoize
+import org.osgeo.proj4j.NotFoundException
+import geotrellis.proj4.io.wkt.Node
+import scala.util.parsing.combinator.RegexParsers
+import scala.util.parsing.combinator._
+
+object WKTParser extends RegexParsers {
+  // private lazy val wkttoepsgcodemap = new Memoize[String, String]
+  private val wktResourcePath = "/geotrellis/proj4/wkt/epsg.properties"
+  private var currentSet: scala.collection.mutable.Set[Any] = scala.collection.mutable.Set.empty
+
+  private def withWktFile[T](f: Iterator[String] => T) = {
+    val stream = getClass.getResourceAsStream(wktResourcePath)
+    try {
+            f(Source.fromInputStream(stream).getLines())
+    } finally {
+            stream.close()
+    }
+  }
+
+  override def skipWhitespace: Boolean = false
+  def symbol: Parser[String] = """[A-Za-z0-9_]+""".r
+
+  def string: Parser[String] = "\"" ~> """[^\"]+""".r <~ "\"" //"""[A-Za-z()/+ 0-9\-&'\,\.\*\?\_]+""".r
+
+  def double: Parser[Double] =("""[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?""".r | """[-+]?[0-9]*\.?[0-9]+""".r) map (_.toDouble)
+
+  def int: Parser[Int] = """[-+]?[0-9]+""".r map (_.toInt)
+
+  def value: Parser[Any] = string | double | int
+
+  def comma: Parser[Any] = ("," ~ " ".?) map {
+    case x ~ _ => None
+  }
+
+  def parameter: Parser[Parameter] = """PARAMETER[""" ~ string ~ comma ~ double ~ """]""" map {
+    case _ ~ name ~ _ ~ value ~ _=> new Parameter(name, value)
+  }
+
+  def authority: Parser[Authority] = """AUTHORITY[""" ~ string ~ comma ~ string ~ """]""" map {
+    case _ ~ x ~ _~ y ~ _ => new Authority(x, y)
+  }
+
+  def spheroid: Parser[Spheroid] = """SPHEROID[""" ~ string ~ comma ~ (double | int) ~ comma ~ double ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ axis ~ _ ~ flattening ~ _ ~ authority ~ _ => new Spheroid(name, axis, flattening, authority)
+  }
+
+  def toWgs: Parser[ToWgs84] = """TOWGS84[""" ~ (double <~ ", ".?).* ~ """]""" map {
```

Comment:
This should use existing `comma` parser instead of `", "` literal. The failing tests are being tripped up by the extra space.